.github/workflows/pr.yml

```diff
@@ -80,7 +83,7 @@ jobs:
     # Build and push image to Google Container Registry
     - name: Build
       run: |-
-        gradle jib --image "$REGISTRY_HOSTNAME/$GKE_PROJECT_ID/imis:$GITHUB_SHA"
```

Comment:
Not sure if it was a good idea to remove the  `--image` argument. Deploy seems not that happy about it, complaining that the container image cannot be found.

Reply:
hmm 🤔 should be covered by https://github.com/ImisDevelopers/1_011_a_infektionsfall_uebermittellung/blob/4bbbf0223e724d640f77212797c0bd44a80419fc/server/build.gradle#L81 and works locally ... 🤔

Reply:
Problem is that this image name is not exactly equal to what the image of the reviewed code line was named previously (note the `imis:` vs. the `imis-server:`). And Deploy expects exactly that name.